### PR TITLE
Cookie Auto: set properties back to ticket when using SessionStore

### PR DESCRIPTION
**Pull request by troydai**
#973

### CODE REVIEW

File: src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 1]
```diff
@@ -188,7 +188,7 @@ private CookieOptions BuildCookieOptions()
                         new ClaimsIdentity(
                             new[] { new Claim(SessionIdClaim, _sessionKey, ClaimValueTypes.String, Options.ClaimsIssuer) },
                             Options.AuthenticationScheme));
-                    ticket = new AuthenticationTicket(principal, null, Options.AuthenticationScheme);
+                    ticket = new AuthenticationTicket(principal, ticket.Properties, Options.AuthenticationScheme);
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: we need to be more selective about the properties we keep in the cookie. only take what you need.

File: src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 2]
```diff
@@ -172,13 +174,22 @@ private CookieOptions BuildCookieOptions()
...
+                    properties.IssuedUtc = _refreshIssuedUtc;
                 }
+
                 if (_refreshExpiresUtc.HasValue)
                 {
-                    ticket.Properties.ExpiresUtc = _refreshExpiresUtc;
+                    properties.ExpiresUtc = _refreshExpiresUtc;
+                }
+
+                var cookieOptions = BuildCookieOptions();
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs]: you can move this back, only the properties change is needed.

**Comment by troydai** [comment]
/cc @Tratcher updated.

As the PR is approved. I'll merge once CI passed.

**Pull request merged**